functions/New-DbaConnectionStringBuilder.ps1

```diff
@@ -134,6 +139,9 @@ function New-DbaConnectionStringBuilder {
             if ($ColumnEncryptionSetting -eq "Enabled") {
                 $builder['Column Encryption Setting'] = "Enabled"
             }
+            if ($pooled) {
```

Comment:
Does this work? If -NonPooledConnection, then $NonPooledConnection is $true, $pooled is $false and $builder['Pooled'] would not be set. I would just set $builder['Pooled'] in any case.

Reply:
haha i dont know, i kinda confused myself. let me test.

Reply:
yeah, i had to fix a keyword but it works (I have a bunch of default params)

![image](https://user-images.githubusercontent.com/8278033/126213379-34cbd1e6-efe6-450d-a218-3bb1b3c968f3.png)